internal/experimental/pusher/ocipusher_test.go

```diff
@@ -14,15 +13,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package registry // import "helm.sh/helm/v3/internal/experimental/registry"
+package pusher
 
 import (
-	"github.com/containerd/containerd/remotes"
+	"testing"
 )
 
-type (
-	// Resolver provides remotes based on a locator
-	Resolver struct {
-		remotes.Resolver
+func TestNewOCIPusher(t *testing.T) {
+	testfn := func(ops *options) {
+		if ops.registryClient == nil {
+			t.Fatalf("the OCIPusher's registryClient should not be null")
+		}
 	}
-)
+
+	NewOCIPusher(testfn)
```

Comment:
what is expected here?  the `NewOCIPusher` returns an error.. do we want to assert nil?  or that the client != nil?  is this call with the expectation of a panic?   This test looks incomplete.

Reply:
Resolved in 241f23d72a53ee578bfaef84ae26ed86b4b5d6d2